crypto/aes/aesopt.h

```diff
@@ -295,26 +295,26 @@ Issue Date: 20/12/2007
     to be set independently for encryption and decryption
 */
 #if 1
-#  define ENC_UNROLL  FULL
+#  define ENC_UNROLL  NONE
```

Comment:
I think here it is meant to change `1` to `0` above instead of swapping the definitions. Creates smaller diff with the upstream.

Reply:
Done (after force-push) in https://github.com/trezor/trezor-firmware/pull/4107/commits/3d2c623d80e6b6595dcc028cfdb2c5b8dd9a1b08.